pulsar-broker-common/pom.xml

```diff
@@ -65,6 +65,23 @@
       <artifactId>jjwt-impl</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>com.microsoft.azure</groupId>
```

Comment:
is this still required ?
in this case we have to deal with LICENSE and NOTICE, and also please declare the dependency as property as you did for auth0

Reply:
> is this still required? 

nope, I used it for some old stuff. Thanks! 😄